templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -62,6 +62,13 @@
             "name": "GALLEON_PROVISION_LAYERS",
             "required": false
         },
+        {
+            "displayName": "Enable ExampleDS datasource",
+            "description": "Enable ExampleDS datasource.",
+            "name": "ENABLE_GENERATE_DEFAULT_DATASOURCE",
```

Comment:
Does this default to true or false in the image now? I don't have any issue with adding the var to the templates, I'm just curious if the default behaviour changed? (The default datasource is h2?)

Reply:
In the image, h2-driver is present. ExampleDS is not. You need to ask for it with ENABLE_GENERATE_DEFAULT_DATASOURCE=true. We set a default value to false, so we will need to document that it needs to be set if needed (we have 2 qs that need it).
When users are using Galleon, then they don't want it by default (no h2 driver in galleon layers). They could use the h2-default-datasource (that will be unsupported for EAP) to still provision h2 and the default ExampleDS datasource.

Reply:
@jfdenise Got it, thanks. +1 from me.